nexus/types/src/external_api/params.rs

```diff
@@ -1259,6 +1331,31 @@ pub struct RouterRouteUpdate {
     pub destination: RouteDestination,
 }
 
+// INTERNET GATEWAYS
+
+/// Create-time parameters for an `InternetGateway`
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
+pub struct InternetGatewayCreate {
+    #[serde(flatten)]
+    pub identity: IdentityMetadataCreateParams,
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
+pub struct InternetGatewayIpPoolCreate {
+    #[serde(flatten)]
+    pub identity: IdentityMetadataCreateParams,
+    pub gateway: NameOrId,
```

Comment:
`internet_gateway_ip_pool_create` already has a query parameter named `gateway`. The good news. I believe the good news is that this field isn't used and can be deleted without consequence. If that's the case (big if), I'm a bit confused as to why clippy isn't barking about it.

Reply:
Fixed in [cf184e6](https://github.com/oxidecomputer/omicron/pull/6475/commits/cf184e6e8dcafb1e83cdd999827bf9e3ee237fae) with corresponding PR in oxidecomputer/oxide.rs#864.